hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocol/OzoneManagerProtocol.java

```diff
@@ -894,6 +895,20 @@ default OpenKeySession createFile(OmKeyArgs keyArgs, boolean overWrite,
   List<OzoneFileStatus> listStatus(OmKeyArgs keyArgs, boolean recursive,
       String startKey, long numEntries) throws IOException;
 
+  /**
+   * Lightweight listStatus API.
+   *
+   * @param keyArgs    Key args
+   * @param recursive  For a directory if true all the descendants of a
+   *                   particular directory are listed
+   * @param startKey   Key from which listing needs to start. If startKey exists
+   *                   its status is included in the final list.
+   * @param numEntries Number of entries to list from the start key
+   * @return list of file status
+   */
+  List<OzoneFileStatusLight> listStatusLight(OmKeyArgs keyArgs,
+      boolean recursive, String startKey, long numEntries) throws IOException;
```

Comment:
`listStatus` methods evolved over time by introducing more and more parameters, and the old methods were kept for backward compatibility.  Since `listStatusLight` is a new part of the API, I guess we don't need multiple versions.  I'd consider removing this one.

Reply:
Thank you @adoroszlai for the review, this is a good point for reducing the redundancy - I have made the required changes.

Reply:
Thanks @tanvipenumudy for updating the patch.  It seems `ClientProtocol` needs similar change.

Reply:
Made the change, missed this in the previous commit, thank you!